src/core/meshcop/tcat_agent.cpp

```diff
@@ -523,6 +533,52 @@ Error TcatAgent::HandleSetActiveOperationalDataset(const Message &aIncomingMessa
 
```

Comment:
The TLV type Set Active Operational Dataset resides in the command class commissioning not command class application.
I should say:
if (!CheckCommandClassAuthorizationFlags(mCommissionerAuthorizationField.mCommissioningFlags,
                                             mDeviceAuthorizationField.mCommissioningFlags, &dataset))

Reply:
fixed.